src/zoid/auth/component.jsx

```diff
@@ -62,93 +72,119 @@ export function getAuthComponent() : ZoidComponent<AuthPropsType> {
                     required:   false
                 },
 
-                clientID: {
-                    type:       'string',
-                    value:      () => getClientID()
+                client_id: {
```

Comment:
Please keep this is as `clientID` per convention. If you need to change the query param name you can use `queryParam: 'client_id'`

Reply:
done